lib/kafka/async_producer.rb

```diff
@@ -69,8 +69,9 @@ class AsyncProducer
     #   buffered messages that will automatically trigger a delivery.
     # @param delivery_interval [Integer] if greater than zero, the number of
     #   seconds between automatic message deliveries.
+    # @param finally [Lambda] will be called on messages remaining on shutdown.
```

Comment:
Just to be explicit it might be a good idea to document that the lambda will take an array